src/app/Model/ValidationError.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text.Json.Serialization;
+
+namespace CSE.Helium.Model
+{
+    public class ValidationError
+    {
+        [JsonPropertyName("code")]
```

Comment:
these shouldn't be necessary - json options are set to camelCase

Reply:
Removed call `JsonPropertyName` references and tested this. This required duplication of the JSON serialization options in the `ValidationProblemDetailsResult` as the settings in `startup.cs` don't apply when you write directly to the Http response (they are bypassed). I tested this initially and had to add the attribute declaration to the classes to get it formatted correctly. Since I now create the options inside the method returning the result anyway, they can be removed.